ee/agent/storage/bbolt/backup.go

```diff
@@ -0,0 +1,225 @@
+package agentbbolt
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/kolide/launcher/ee/agent/types"
+	"go.etcd.io/bbolt"
+)
+
+const (
+	backupInitialDelay         = 10 * time.Minute
+	backupInterval             = 1 * time.Hour
```

Comment:
I think we can even lower this. 6 hours, maybe? The only things we really care about are keys and eventually registrations.